spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/TableIdent.scala

```diff
@@ -0,0 +1,6 @@
+package org.apache.spark.sql.cassandra
+
+
+/** Store table identity including table name, keyspace name and option cluster name */
+case class TableIdent(table: String, keyspace: String, cluster: Option[String] = None)
```

Comment:
Why keyspace is a String is trying to force user to specify the keyspace/database name. We can add a default keyspace name of "default"